templates/content/dymaptic.GeoBlazor.Template.WebAssembly/dymaptic.GeoBlazor.Template.WebAssembly.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="dymaptic.GeoBlazor.Core" Version="2.5.3" />
```

Comment:
I wonder if there is a way to say use the latest, does * work?

Reply:
Part of our automated build in github actions automatically updates this when I change the version # in the props file.